src/lib/containers/entity/page/title_bar/TitleBarProperties.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react'
+import { Box, Divider, Link, Stack, Typography } from '@mui/material'
+import { useGetEntityBundle } from '../../../../utils/hooks/SynapseAPI'
+import AddConditionsForUseButton from '../../../access_requirements/AddConditionsForUseButton'
+import { useSynapseContext } from '../../../../utils/SynapseContext'
+import { useGetEntityTitleBarProperties } from './useGetEntityTitleBarProperties'
+
+export type TitleBarPropertiesProps = {
+  entityId: string
+  versionNumber?: number
+  /* Callback invoked when an ACT member clicks "Add Conditions for Use". */
+  onActMemberClickAddConditionsForUse: () => void
+}
+
+type PropertyProps = {
+  title: string
+  value: React.ReactNode
+}
+function Property(props: PropertyProps) {
+  const { title, value } = props
+  return (
+    <tr>
+      <td style={{ paddingRight: '1em' }}>
+        <Typography
+          component={'span'}
+          variant={'smallText1'}
+          sx={{
+            fontWeight: 700,
+          }}
+        >
+          {title}
+        </Typography>
+      </td>
+      <td>
+        <Typography component={'span'} variant={'smallText1'}>
+          {value}
+        </Typography>
+      </td>
+    </tr>
+  )
+}
+
+/**
+ * The TitleBarProperties component displays a tabular view of the entity metadata on the Entity page.
+ */
+export default function TitleBarProperties(props: TitleBarPropertiesProps) {
+  const { entityId, versionNumber, onActMemberClickAddConditionsForUse } = props
+  const { isInExperimentalMode } = useSynapseContext()
+
+  // We don't need the entire bundle, but it's fetched for the rest of the title bar and useGetEntityTitleBarProperties below, so the cache will be hot.
+  const { data: bundle } = useGetEntityBundle(entityId, versionNumber)
+
+  // Actual entity data is fetched and transformed in this custom hook
+  const properties = useGetEntityTitleBarProperties(entityId, versionNumber)
+
+  const [showAllProperties, setShowAllProperties] = useState(false)
+
+  return (
+    <Box sx={{ padding: '20px 40px' }}>
+      <Stack
+        direction={'row'}
+        justifyContent={'space-between'}
+        alignItems={'flex-start'}
+      >
+        <Stack
+          direction="row"
+          alignItems={'flex-start'}
+          divider={<Divider orientation="vertical" flexItem />}
+          spacing={2}
+        >
+          {/* Always show the first 4 properties*/}
+          <table>
```

Comment:
Using multiple tables was the easiest way to accomplish the behavior in the design. My hope is that it doesn't cause significant accessibility issues.